jobs/bosh-dns/spec

```diff
@@ -174,3 +174,6 @@ properties:
   logging.format.timestamp:
     description: "Format for the timestamp in the component logs.  Valid values are 'rfc3339' and 'deprecated'."
     default: "rfc3339"
+
+  logging.tags:
+      description: "Tags for logging to provide individual configuration to the logger."
```

Comment:
Can we state here that this is optional and what happens if not configured. This part `to provide individual configuration to the logger.` says a little bit what will happen but it is not completely clear.